kore/src/Kore/Repl/Data.hs

```diff
@@ -400,21 +401,19 @@ helpText =
     "Available commands in the Kore REPL: \n\
     \help                                     shows this help message\n\
     \claim [n|<name>]                         shows the nth claim, the claim with\
-                                              \ <name> or if used without args\
-                                              \ shows the currently focused claim\
-                                              \ in the form: LHS => (modality) RHS \n\
+                                              \ <name> or (default) the currently\
```

Comment:
The description of each command should be one line. We can include a link to the wiki page at the end of the help message.